src/components/molecules/widgets/AccidentCountByCarTypeWidget.tsx

```diff
@@ -27,22 +27,26 @@ const AccidentCountByCarType: FC<IProps> = ({ data, segmentText }) => {
     percentage_segment: Math.round(i.percentage_segment),
     percentage_country: Math.round(i.percentage_country),
   }));
+  const xLabels = [PERCENTAGE_SEGMENT,PERCENTAGE_COUNTRY];
+  const xNames = [t('widgets.countByCarType.percentage segment'),t('widgets.countByCarType.percentage country')]
```

Comment:
how's the string working here? There's a space before the last word. Is this on purpose?

Reply:
works in translation mechanism but fixed anyway to have separator